Makefile

```diff
@@ -183,7 +184,12 @@ test: vendor $(ADDON_ASSET_FILE)
 .PHONY: integration
 integration: $(MINISHIFT_BINARY)
 	mkdir -p $(INTEGRATION_TEST_DIR)
-	go test -timeout 3600s $(REPOPATH)/test/integration --tags=integration -v -args --test-dir $(INTEGRATION_TEST_DIR) --binary $(MINISHIFT_BINARY) $(GODOG_OPTS)
+	go test -timeout $(TIMEOUT) $(REPOPATH)/test/integration --tags=integration -v -args --test-dir $(INTEGRATION_TEST_DIR) --binary $(MINISHIFT_BINARY) $(GODOG_OPTS)
+
+.PHONY: integration_all
+integration_all: $(MINISHIFT_BINARY)
+	mkdir -p $(INTEGRATION_TEST_DIR)
+	go test -timeout $(TIMEOUT) $(REPOPATH)/test/integration --tags=integration -v -args --test-dir $(INTEGRATION_TEST_DIR) --binary $(MINISHIFT_BINARY) -tags ~coolstore
```

Comment:
I am fine with either as we expect that user will provide the value in case the test does not complete in 3600s.

Reply:
@LalatenduMohanty right however i see some genuine reason behind @agajdosi concern. Through ci all features are being executed. Not sure the default timeout might be sufficient. IMO to avoid time out failure form ci side we either we can increase the common default timeout for all integration target around ```TIMEOUT ?= 5400s```or pass the variable TIMEOUT=7200s in ci file.
I fine with timeout set by user if test doesn't complete in default 3600s

Reply:
Lets keep the timeout 3600s as we discussed now.